.github/Dockerfile-release

```diff
@@ -0,0 +1,9 @@
+FROM busybox:latest as build
```

Comment:
When is this used vs the Dockerfile in the root?
https://github.com/pomerium/pomerium/pull/586/files#diff-3254677a7917c6c01f55212f86c57fbf

Reply:
Dockerfile at the root is for just a normal container build that someone might use for testing.  The release ones are just for goreleaser since they require a build to be run externally.

I think the normal build just needs to cover x86_64 for testing, so I removed the other archs.  We could maybe add back in ARM64 if you think that's a good idea.

Reply:
> Dockerfile at the root is for just a normal container build that someone might use for testing. The release ones are just for goreleaser since they require a build to be run externally.

Got it. Makes complete sense. 
> We could maybe add back in ARM64 if you think that's a good idea.
Nah, I think we are good with a generic x64 for now. Maybe add a debug build per #584 in a follow up PR.

Reply:
That top level one could just include user tools.  I'll follow up on that.